### PR TITLE
[PR] Display

### PR DESCRIPTION
**Pull request by philcable**
A shortcode for a front-end view very similar to those found on https://screens.wsu.edu/alumniwallofhonor/.

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

File: includes/class-wsuwp-alumni-awards.php [hunk 1]
```diff
@@ -271,4 +272,142 @@ public function admin_enqueue_scripts( $hook_suffix ) {
...
+		);
+
+		$atts = shortcode_atts( $defaults, $atts );
+
+		// The `award_slug` attribute must be set.
+		if ( '' === $atts['award_slug'] ) {
+			return '<!-- No award_slug attribute set -->';
+		}
+
+		wp_enqueue_style( 'alumni-awards', plugins_url( 'css/shortcode.css', dirname( __FILE__ ) ) );
```
Comment (jeremyfelt) [inline includes/class-wsuwp-alumni-awards.php]: I'd add a version here to make it easier to break cache in the future if necessary.

File: includes/class-wsuwp-alumni-awards.php [hunk 2]
```diff
@@ -271,4 +272,142 @@ public function admin_enqueue_scripts( $hook_suffix ) {
...
+		wp_enqueue_script( 'alumni-awards', plugins_url( 'js/shortcode.min.js', dirname( __FILE__ ) ), array( 'jquery' ) );
```
Comment (jeremyfelt) [inline includes/class-wsuwp-alumni-awards.php]: Version here as well (see above).

**Review by jeremyfelt — CHANGES_REQUESTED** [review]
Looks good, just one quick change. :)

**Comment by philcable** [comment]
@jeremyfelt Excellent suggestion! Updated - #reviewmerge?

**Review by jeremyfelt — APPROVED** [review]

**Comment by jeremyfelt** [comment]
:zap:

**Pull request merged**